setup.py

```diff
@@ -458,6 +455,11 @@ def pxd(name):
            'sources': ['pandas/src/datetime/np_datetime.c',
                        'pandas/src/datetime/np_datetime_strings.c',
                        'pandas/src/period.c']},
+    period=dict(pyxfile='period',
```

Comment:
I just wanted them all to be consistent, using ``{...}`` for dict is what I recall